doc/ref/plotting_options/interactivity.ipynb

```diff
@@ -0,0 +1,286 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(title='Scatter with hover tool (Default)', **opts)\n",
+    "plot2 = df.hvplot.scatter(hover=False, title='Scatter without hover tool', **opts)\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30839bfb-e814-42d8-bac6-d61b5b84cead",
+   "metadata": {},
+   "source": [
+    "(options-hover_cols)=\n",
+    "## `hover_cols`\n",
+    "\n",
+    "Specifies additional columns from the dataset to be shown in the hover tooltip.\n",
+    "- Accepts a list of column names, a single column name as a string, or 'all' to include all available columns.\n",
+    "- When set to 'all', it includes index columns only if `use_index=True`.\n",
+    "- Ignored for `datashade=True` plots, as those do not support interactivity.\n",
+    "\n",
+    "::: {note} \n",
+    "`hover_cols` complements the default dimensions shown in the tooltip but does not override them.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88a10e38-37e7-4fc1-941f-850acd2d9486",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.head(3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "366fb132-4103-4e1c-843d-df90fed364eb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(title='Default hover_cols', **opts)\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    hover_cols=['sex', 'body_mass_g'], title='Additional hover columns', **opts\n",
+    ")\n",
+    "plot3 = df.hvplot.scatter(\n",
+    "    hover_cols='all', title='All hover columns', **opts\n",
+    ")\n",
+    "\n",
+    "plot4 = df.hvplot.scatter(\n",
+    "    hover_cols='all',\n",
+    "    use_index=False,\n",
+    "    title='All hover columns without index',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "(plot1 + plot2 + plot3 + plot4).cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2560fd13-ccc7-4be1-805f-08f60912c487",
+   "metadata": {},
+   "source": [
+    "Hover on the points in each of the examples to see the displayed hover information."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e2e2707-5c15-4ae5-bc4d-dc3f14bee454",
+   "metadata": {},
+   "source": [
+    "(options-hover_formatters)=\n",
+    "## `hover_formatters`\n",
```

Comment:
The example in this section is not working as expected. In fact, I'm not entirely sure of the accuracy of what is written in this `hover_formatters` section.

Reply:
Can you please open an issue if there's a bug?

Reply:
It was a bug 😄 

I've opened an issue [here](https://github.com/holoviz/holoviews/issues/6589)

Reply:
Can you ask Andrew if he has an example where `hover_formatters` is useful?